Mathlib/Algebra/Module/ZLattice/Covolume.lean

```diff
@@ -96,6 +116,217 @@ theorem covolume_eq_det {ι : Type*} [Fintype ι] [DecidableEq ι] (L : Submodul
   ext1
   exact b.ofZLatticeBasis_apply ℝ L _
 
-end Real
+theorem covolume_eq_det_inv {ι : Type*} [Fintype ι] [DecidableEq ι] (L : Submodule ℤ (ι → ℝ))
```

Comment:
Can you use `Finite` instead of `Fintype`?

Reply:
Unfortunately, the statement does not type check with `Finite` instead of `Fintype`